README.md

```diff
@@ -65,42 +63,59 @@ List of dependencies:
 6. gcc (>=5.4.0)
 7. faiss-gpu (>=1.4.0) - To install with conda: ```conda install -c pytorch faiss-gpu cuda92```
 
-### Setup steps
+### Installing from Source:
 
-To clone:
+Once dependencies are present, follow the steps below:
 
+1. Clone the repository.
+```bash
+$ git clone --recurse-submodules https://github.com/rapidsai/cuml.git
```

Comment:
Is `--recurse-submodules` is the same as the same as clone + `git submodule update --init --recursive`?

Reply:
Yes this is the same as doing:
```
git clone ...
git submodule update --init --recursive
```